lib/git_reflow/commands/review.rb

```diff
@@ -5,16 +5,45 @@
   c.arg_name '[destination_branch] - the branch you want to merge your feature branch into'
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
+  c.switch [:e, :edit]
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    commit_msg_file = '/tmp/git_reflow_pr_msg'
```

Comment:
I don't think we can guarantee that every system will have a `/tmp` directory.  can we use the `.git` directory and create another file like git does with the commit messages?  maybe `.git/GIT_REFLOW_PR_MSG`